lib/merge-config.js

```diff
@@ -211,7 +211,7 @@ function mergeConfig(options, context, gitRawCommitsOpts, parserOpts, writerOpts
         },
         parserOpts);
 
-      if (hostOpts.referenceActions && parserOpts) {
+      if (!parserOpts.hasOwnProperty('referenceActions') && hostOpts.referenceActions) {
```

Comment:
`if (_.isEmpty(parserOpts. referenceActions) && hostOpts. referenceActions)`
Should be the same. but to be consistent with line 218